apps/site/lib/site_web/views/customer_support_view.ex

```diff
@@ -45,6 +47,13 @@ defmodule SiteWeb.CustomerSupportView do
     )
   end
 
+  @spec subject_map([Message.service_option_with_subjects()]) ::
```

Comment:
Dialyzer seems unhappy with this spec. Should the output be a map instead of a tuple maybe?